### PR TITLE
Adds notes about incompatibility between 1.X and 2.X configurations.

### PR DESCRIPTION
**Pull request by SantoDE**
### What does this PR do?

Adds a note to the v2 docs about incompatible behaviour and adds a checkbox to the issue templates about that fact to check.

### Motivation

Make it clear

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ For end-user related support questions, please refer to one of the following:
...
 -->
 
+### Did you try using a 1.7.x configuration for the version 2.0?
+
+- [ ] Yes
+- [ ] No
+
+<!--
+
+If you just checked that box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```
Comment (Ullaakut) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 2]
```diff
@@ -20,6 +20,17 @@ For end-user related support questions, please refer to one of the following:
...
 Bug
 
+### Did you try using a 1.7.x configuration for the version 2.0?
+
+- [ ] Yes
+- [ ] No
+
+<!--
+
+If you just checked that box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```
Comment (Ullaakut) [inline .github/ISSUE_TEMPLATE/Bug_report.md]:
```suggestion
If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```

File: docs/content/getting-started/configuration-overview.md [hunk 3]
```diff
@@ -29,6 +29,10 @@ Traefik gets its _dynamic configuration_ from [providers](../providers/overview.
  
 ## The Static Configuration
 
+!!! warning "Uncompatible Static Configuration"
+    Please be aware, that the old configuration of v1.7.X is NOT compatible with the v2.X config as of now.
```
Comment (Ullaakut) [inline docs/content/getting-started/configuration-overview.md]:
```suggestion
    Please be aware that the old configuration of v1.7.X is NOT compatible with the v2.X config as of now.
```

File: docs/content/getting-started/configuration-overview.md [hunk 4]
```diff
@@ -29,6 +29,10 @@ Traefik gets its _dynamic configuration_ from [providers](../providers/overview.
  
 ## The Static Configuration
 
+!!! warning "Uncompatible Static Configuration"
+    Please be aware that the old configuration of v1.7.X is NOT compatible with the v2.X config as of now.
```
Comment (dtomcej) [inline docs/content/getting-started/configuration-overview.md]:
```suggestion
    Please be aware that the old configurations for Traefik v1.X are NOT compatible with the v2.X config as of now.
```

File: docs/content/getting-started/configuration-overview.md [hunk 5]
```diff
@@ -29,6 +29,10 @@ Traefik gets its _dynamic configuration_ from [providers](../providers/overview.
...
+    If you're testing out v2, please create a fresh configuration.
```
Comment (dtomcej) [inline docs/content/getting-started/configuration-overview.md]:
```suggestion
    If you're testing out v2, please ensure you are using a v2 configuration.
```

File: .github/ISSUE_TEMPLATE/Bug_report.md [hunk 6]
```diff
@@ -20,6 +20,17 @@ For end-user related support questions, please refer to one of the following:
...
 Bug
 
+### Did you try using a 1.7.x configuration for the version 2.0?
+
+- [ ] Yes
+- [ ] No
+
+<!--
+
+If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```
Comment (dtomcej) [inline .github/ISSUE_TEMPLATE/Bug_report.md]:
```suggestion
If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -16,6 +16,17 @@ For end-user related support questions, please refer to one of the following:
...
 -->
 
+### Did you try using a 1.7.x configuration for the version 2.0?
+
+- [ ] Yes
+- [ ] No
+
+<!--
+
+If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.7.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```
Comment (dtomcej) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
If you just checked the "Yes" box, be aware that this is probably not a bug. The configs between 1.X and 2.X are NOT compatible. Please have a look here https://docs.traefik.io/v2.0/getting-started/configuration-overview/.
```

File: docs/content/getting-started/configuration-overview.md (hunk 4 above)
Comment (Ullaakut) [reply docs/content/getting-started/configuration-overview.md]: I'd suggest also replacing `config` with `configuration` for consistency

File: .github/ISSUE_TEMPLATE/Bug_report.md (hunk 6 above)
Comment (Ullaakut) [reply .github/ISSUE_TEMPLATE/Bug_report.md]: I'd suggest also replacing `configs` with `configuration` for consistency

File: .github/ISSUE_TEMPLATE.md (hunk 7 above)
Comment (Ullaakut) [reply .github/ISSUE_TEMPLATE.md]: Same here

File: docs/content/getting-started/configuration-overview.md [hunk 8]
```diff
@@ -29,6 +29,10 @@ Traefik gets its _dynamic configuration_ from [providers](../providers/overview.
  
 ## The Static Configuration
 
+!!! warning "Incompatible Static Configuration"
```
[the remark this answers is not in the document]
Comment (SantoDE) [reply docs/content/getting-started/configuration-overview.md]: Done

**Pull request merged**